app/models/submission/validations_by_template_name.rb

```diff
@@ -32,10 +40,26 @@ def apply_additional_validations_by_template_name
     case submission_template_name
     # this validation is for the scRNA pipeline cDNA submission
     when SCRNA_CORE_CDNA_PREP_GEM_X_5P
-      validate_consistent_column_value(HEADER_NUM_SAMPLES)
+      validate_consistent_column_value(HEADER_NUMBER_OF_POOLS)
       validate_consistent_column_value(HEADER_CELLS_PER_CHIP_WELL)
+      validate_samples_per_pool_for_labware
+      if Flipper.enabled?(:y24_429_enable_check_feasibility_of_cdna_prep_submission)
```

Comment:
just checking the feature flag is still valid if we're merging this now

Reply:
It's OK, I've added a note to remember to turn it on when we deploy. We should make a clean up story (if haven't already) for getting rid of it.